.gitlab-ci/stages/data.yml

```diff
@@ -1,7 +1,9 @@
 .build_and_push_stage: &build_and_push_stage
   extends: .base_docker_image_stage
-  stage: "Data"
+  stage: "Post Registration"
```

Comment:
I think we could move it to `Registration` stage now we can use `needs`, no ?

Reply:
it was done in a separate stage because I need to wait after nlp registration was completed but with  `needs`I think it no more usefull